AUTHORS

```diff
@@ -92,6 +92,7 @@ Evan Kepner
 Fabien Zarifian
 Fabio Zadrozny
 Feng Ma
+Fernando Mezzabotta Rey
```

Comment:
I'm not sure if this the thing to do when back-porting things. If it's not, let me know! Just following the steps-to-contribute 🙂